### PR TITLE
Add sub-band to README

### PR DESCRIPTION
**Pull request by htdvisser**
#### Summary

what the title says

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,12 @@ rx2-channel:                   # Rx2 channel (optional)
   max-data-rate: 5
   radio: 0
 rx2-default-data-rate: 0       # Default data rate index of Rx2 (optional)
-max-eirp: 29.15                # Maximum EIRP to use regardless of band settings (optional)
+max-eirp: 29.15                # Maximum EIRP for this frequency plan (optional)
+sub-bands:                     # Sub-bands override the duty cycle and maximum EIRP for frequency ranges.
+- min-frequency: 868000000     # Minimum frequency (Hz)
+  max-frequency: 868600000     # Maximum frequency (Hz)
+  duty-cycle: 0.01             # Duty cycle for this sub-band (optional)
```
Comment (johanstokking) [inline README.md]: Add that if it's not set (or `0`), duty-cycle does not imply (we imply `1` in fact)

File: README.md [hunk 2]
```diff
@@ -63,7 +63,12 @@ rx2-channel:                   # Rx2 channel (optional)
   max-data-rate: 5
   radio: 0
 rx2-default-data-rate: 0       # Default data rate index of Rx2 (optional)
-max-eirp: 29.15                # Maximum EIRP to use regardless of band settings (optional)
+max-eirp: 29.15                # Maximum EIRP for this frequency plan (optional)
+sub-bands:                     # Sub-bands override the duty cycle and maximum EIRP for frequency ranges.
```
Comment (johanstokking) [inline README.md]: nit; Remove trailing `.`

File: README.md [hunk 3]
```diff
@@ -63,7 +63,12 @@ rx2-channel:                   # Rx2 channel (optional)
...
+- min-frequency: 868000000     # Minimum frequency (Hz)
+  max-frequency: 868600000     # Maximum frequency (Hz)
+  duty-cycle: 0.01             # Duty cycle for this sub-band (optional)
+  max-eirp: 16.15              # Maximum EIRP for this sub-band (optional)
```
Comment (johanstokking) [inline README.md]: It's officially under `band-id` (so in the top)

Comment (htdvisser) [reply README.md]: What do you mean?

Comment (johanstokking) [reply README.md]: Comment on the wrong line, but I meant to move `sub-bands` to right under `band-id`

File: README.md [hunk 4]
```diff
@@ -12,6 +12,11 @@ Frequency plan are defined in YAML files. Most settings in the frequency plan ar
 
 ```yml
 band-id: BAND_ID               # ID of the band
+sub-bands:
+- min-frequency: 868000000     # Minimum frequency (Hz)
+  max-frequency: 868600000     # Maximum frequency (Hz)
+  duty-cycle: 0.01             # Duty cycle for this sub-band (optional; default: 1)
+  max-eirp: 16.15              # Maximum EIRP for this sub-band (optional; takes precedence over )
```
Comment (johanstokking) [inline README.md]: over what?

File: README.md [hunk 5]
```diff
@@ -12,6 +12,11 @@ Frequency plan are defined in YAML files. Most settings in the frequency plan ar
 
 ```yml
 band-id: BAND_ID               # ID of the band
+sub-bands:
+- min-frequency: 868000000     # Minimum frequency (Hz)
+  max-frequency: 868600000     # Maximum frequency (Hz)
```
Comment (johanstokking) [inline README.md]: These frequencies are inclusive

**Comment by johanstokking** [comment]
Pushed some changes based on my review above to `master`

**Comment by htdvisser** [comment]
Yes I saw that, maybe we should also add this "inclusive" to the other min-frequency and max-frequency we have in the example

**Pull request merged**